### PR TITLE
Installation summary: fix NVME partition formatting

### PR DESCRIPTION
**Pull request by jpnurmi**
"nvme0n11" vs. "nvme0n1p1"

For historical reasons, we used "{disk}{partition}" because it used to be something like "{disk} #{partition}". However, this does not result in the correct formatting with NVME disks where there needs to be an extra "p". It's not possible to change the translations on the release week but luckily all existing translations have kept it as "{disk}{partition}" so we can safely pass an empty disk argument for the time being, and remove the unused argument later. The "Partition.path" property from subiquity provides the correct value, and we have a "sysname" convenience wrapper for that to take the filename part of the path.

| Before | After |
|---|---|
| Screenshot from 2023-04-17 14-26-37 | Screenshot from 2023-04-17 14-25-27 |

Fixes: #1819

### CODE REVIEW

**Review by seb128 — APPROVED** [review]
Seems fine to me, as for the other one we should have a +1 from release team before landing

**Review by d-loose — APPROVED** [review]
LGTM :+1:

**Comment by jpnurmi** [comment]
Agreed with @sil2100

**Pull request merged**
